galite-tests/src/test/kotlin/org/kopi/galite/tests/form/FormTests.kt

```diff
@@ -119,6 +120,10 @@ class TestBlock : FormBlock(1, 1, "Test", "Test block") {
       priority = 1
     }
   }
+  val image = visit(domain = Domain<Image>(20), position = at(1, 1)) {
+    label = "image"
+    help = "The user image"
+  }
```

Comment:
you need to do relation between this and our Model tho show the Image ,

for help : you can use the form "ArticlesStock.vf" under SRD5, and you can see the representation of the image with this query under DUC_DEV_0 : "select photo from articlesStock where photo is not null"